.gitattributes

```diff
@@ -11,5 +16,23 @@ phpunit.xml.dist   export-ignore
 CHANGELOG.md       export-ignore
 CONTRIBUTING.md    export-ignore
 README.md          export-ignore
+phpstan.neon.dist  export-ignore
+
+#
+# Auto detect text files and perform LF normalization
+# http://davidlaing.com/2012/09/19/customise-your-gitattributes-to-become-a-git-ninja/
+#
+* text=auto
+
+#
+# The above will handle all files NOT found below
+#
+*.md text
+*.php text
+*.inc text
+
+#
+# Pretty color the neon files
+#
 *.neon             linguist-language=YAML
 *.neon.dist        linguist-language=YAML
```

Comment:
linguist got smarter!
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L3607-L3616
Please remove this block.